README.md

```diff
@@ -19,11 +19,6 @@ The application suite consists of three core components:
 
 ## Releases
 
-[Release 5.3.5](https://github.com/google/earthenterprise/releases/tag/5.3.5-1610.20)
```

Comment:
5.3.5.1 did not have these before ?

Reply:
No, it didn't. It appears we need to get some updates from `release_5.3.5.0` that happened after this branch was made.